src/commonmark/en/dhis2_android_implementation_guidelines_INDEX.md

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
You have a yaml header section included in the index file from the adroid repository, so you should not include anything before the `!SUBMODULE` directive (i.e. remove everything before line 6)

Reply:
Sorry! I see you already made that change!